File: app/code-examples/flood-threshold.js
const response = await fetch(
  'https://api-test.openepi.io/flood/threshold?' +
    new URLSearchParams({
      lon: '33.575897',
      lat: '-1.375532',
    })
);
const json = await response.json();

// Get the 2-year return period threshold
const threshold2y = json.queried_location.features[0].properties.threshold_2y;

console.log(`2-year return period threshold in m^3/s: ${threshold2y} m^3/s`);

File: app/code-examples/flood-threshold.py
from httpx import Client

with Client() as client:
    response = client.get(
        url="https://api-test.openepi.io/flood/threshold",
        params={"lon": 33.575897, "lat": -1.375532},
    )

    # Get the 2-year return period threshold
    json = response.json()
    threshold_2y = json["queried_location"]["features"][0]["properties"]["threshold_2y"]

    print(f"2-year return period threshold: {threshold_2y} m^3/s")
